.github/workflows/ateam.yml

```diff
@@ -26,6 +26,13 @@ jobs:
           chmod 600 ~/.ssh/id_rsa
         shell: bash
 
+      - name: Configure AWS key
+        run: |
+          mkdir -p -m 700 ~/.aws/
+          echo "${{ secrets.AWS_CREDENTIALS }}" > ~/.aws/credentials
```

Comment:
Not sure we want to write the AWS creds in a file. This can cause a potential security breach if the machine is compromised or misused by others.

Reply:
You might want to consider using environment variables instead

Reply:
As far as I know, a configuration file with the secret key is automatically created on the server at ~/.aws.
I tried to protect it with `chmod 700` sp that only thet root user can access it

Reply:
Not sure if that what happens when you creds are consumed from environment variables.

Reply:
Done